frontend/src/components/NavigationBar.jsx

```diff
@@ -8,23 +8,23 @@ import {
   Button
 } from 'reactstrap';
 import PollingStatus from './PollingStatus';
-import { chartSizeOptions, pollingOptions, CHAINERUI_VERSION } from '../constants';
+import { chartSizeOptions, pollingOptions, logsLimitOptions, CHAINERUI_VERSION } from '../constants';
 
 
 const RESULT_NAME_ALIGN_LEFT = 'result-name-align-left';
 const RESULT_NAME_ALIGN_RIGHT = 'result-name-align-right';
 
-const createPollingOptionElems = (options) => [
-  ...options.map((option) => (
-    <option key={option.id} value={option.value}>{option.name}</option>
-  ))
-];
+const createPollingOptionElems = (options) => options.map((option) => (
+  <option key={option.id} value={option.value}>{option.name}</option>
+));
 
-const createVisualizerSizeOptionElems = (options) => [
-  ...options.map((option) => (
-    <option key={option.id} value={option.id}>{option.name}</option>
-  ))
-];
+const createVisualizerSizeOptionElems = (options) => options.map((option) => (
+  <option key={option.id} value={option.id}>{option.name}</option>
+));
+
+const createLogsLimitOptionElems = (options) => options.map((option) => (
+  <option key={option.id} value={option.value}>{option.name}</option>
+));
```

Comment:
This method looks like `createPollingOptionElems` .

If you want to reuse it, you need to change the method name. (I think that I don't need to reuse.)